hudi-flink/src/main/java/org/apache/hudi/streamer/HoodieFlinkStreamer.java

```diff
@@ -109,7 +109,7 @@ public static void main(String[] args) throws Exception {
         .transform(
             "bucket_assigner",
             TypeInformation.of(HoodieRecord.class),
-            new KeyedProcessOperator<>(new BucketAssignFunction<>(conf)))
+                new BucketAssignOperator<>(new BucketAssignFunction<>(conf)))
         .setParallelism(conf.getInteger(FlinkOptions.BUCKET_ASSIGN_TASKS))
```

Comment:
Nice catch, can we fix the indentation ? And there is another PR same with this, can we close that ?

Reply:
yes

Reply:
> Nice catch, can we fix the indentation ? And there is another PR same with this, can we close that ?

Is there anything wrong with the code I submitted